File: backend/voting-app/src/main/java/com/tu/votingapp/dto/response/referendum/ReferendumVoteResponseDTO.java
package com.tu.votingapp.dto.response.referendum;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReferendumVoteResponseDTO {
    private Long id;
    private Long userId;
    private Long referendumId;
    private Long optionId;
    private LocalDateTime voteTimestamp;
}

File: frontend/android/app/src/main/java/com/example/android/dto/response/referendum/ReferendumVoteResponseDTO.kt
package com.example.android.dto.response.referendum

import kotlinx.serialization.Serializable

@Serializable
data class ReferendumVoteResponseDTO(
    val id: Long,
    val userId: Long,
    val referendumId: Long,
    val optionId: Long,
    val voteTimestamp: String
)